__tests__/app.test.js

```diff
@@ -189,5 +176,76 @@ describe("NC News", () => {
           expect(msg).toBe("Bad request");
         });
     });
+
+    test('POST 201: Responds with status 201 and new comment when comment successfully added.', () => {
+      return request(app)
+        .post('/api/articles/2/comments')
+        .send({
+          username: 'lurker',
+          body: 'Buy stuff at https://dodgy-site.com'
+        })
+        .expect(201)
+        .then(({ body: { comment } }) => {
+          expect(comment).toMatchObject({
+            comment_id: 19,
+            author: 'lurker',
+            body: 'Buy stuff at https://dodgy-site.com',
+            article_id: 2,
+            votes: 0
+          });
+        });
+    });
+
+    test('POST 404: Responds with status 404 when referenced article does not exist.', () => {
```

Comment:
Great error testing here. It would also be beneficial to add a test for an invalid id eg `/api/articles/invalidid/comments`